Bottom: 71c9dbbf72fc5386020972c46396c46f7739365f
Top:    3e44a56d1b6ea583c1553482c77c71cc8c9248fa
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 17:19:41 -0500

Refresh of buildbot-master

---
diff --git a/ansible-playbook/group_vars/all/buildbot.yml b/ansible-playbook/group_vars/all/buildbot.yml
index 29d7c9f8..c918df8f 100644
--- a/ansible-playbook/group_vars/all/buildbot.yml
+++ b/ansible-playbook/group_vars/all/buildbot.yml
@@ -1,5 +1,5 @@
 ---
-buildbot_version: 1.1.1
+buildbot_version: 1.1.2
 buildbot_worker_port: 9989
 buildbot_worker_ntel: worker-ntel
 buildbot_worker_ptxdist: worker-ptxdist
